src/main/java/com/ue/player/impl/EconomyPlayerImpl.java

```diff
@@ -445,12 +445,20 @@ private void loadBankAccount() {
 		} else {
 			// new loading
 			String iban = config.getString(getName() + ".Iban");
+			// at this code step if the iban is null the player has no existing account, make a new one
+			// safely handles a state where a player name has no account information causing an NPE
+			if (iban == null) {

```

Comment:
I'm not actually certain what causes it but the reason this was an issue is because the PlayerFile I had generated from running 1.2.5f on my server contained usernames for players but those player usernames didn't have any actual data attributed to them. The entire "Player:" section of the file was missing for those users. This would cause the code to reach a point where it had already checked to see if the player had any account information by checking the top of the file for the player's username in the list. Because the name was present it would then assume the player had actual account information but would attempt to parse it and end up with non-existent null data. The simple additional null check effectively catches this edge case with corrupted data and handles it cleanly by just bringing that user up to date.